sources/packages/backend/apps/queue-consumers/src/processors/schedulers/base-scheduler.ts

```diff
@@ -47,9 +49,23 @@ export abstract class BaseScheduler<T> implements OnApplicationBootstrap {
    * any old cron job delete it and add the new job to the queue.
    */
   async onApplicationBootstrap(): Promise<void> {
+    const isFulltimeAllowed = this.configService.isFulltimeAllowed;
```

Comment:
As this is a temporary solution until we go full time, we can construct the config service like this which will avoid the changes in all other scheduler processor files. Revert would be easy.

![image](https://github.com/bcgov/SIMS/assets/54600590/b5e17856-6721-4aa3-b897-8b1d6e625b81)

Reply:
I would be ok if we add a clear comment that this is not inject with the intention of being removed once full-time is release.